plugins/split-yaml.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: split-yaml
+spec:
+  version: v0.1.0
+  homepage: https://github.com/nathforge/kubectl-split-yaml
+  shortDescription: Split Kubernetes YAML output into one file per resource.
```

Comment:
It's OK, but this is more concise :
> Split YAML output into one file per resoure

This field is usually truncated at ~50 chars, so it's better to stay below that threshold

Reply:
Good call, done.